mqt/qecc/cc_decoder/test_square_octagon_color_code.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+from square_octagon_color_code import SquareOctagonColorCode
+
+
+def test_d3():
+    d3_layout = SquareOctagonColorCode(3)
+
+    assert d3_layout.data_qubits == {(0, 1), (2, 1), (6, 1), (3, 2), (5, 2), (3, 4), (5, 4)}
+    assert d3_layout.ancilla_qubits == {(4, 0), (1, 3), (4, 3)}
+
+
+def test_number_of_qubits():
+    for d in range(3, 16, 2):
```

Comment:
I'd recommend making this a parametrised test instead of a loop over the distances.